index.html

```diff
@@ -50,7 +52,47 @@
 				</div>
 			</div>
 		</div>
-
 	</header>
+	<main>
+		<div class="hero container">
+				<div class="hero__item new-label">
+					<div class="hero__wrap-btn"><h2>Материалы</h2><a href="#">на любой вкус</a></div>
+					<div class="hero__img"><img src="images/paint.png" alt="Материалы"></div>
```

Comment:
Please create sprite for all these icons and use class for that something like hero__img--paint